crt_portal/static/js/dashboards-index-page.js

```diff
@@ -0,0 +1,182 @@
+                <div class="usa-card__footer">
+                    <p class="font-body-3xs">
+                        Last Modified: ${new Date(dashboardData.last_modified).toLocaleString(
+                          'en-US',
+                          { hour12: false }
+                        )}
+                    </p>
+                </div>
+            </div>
+        `;
+    dashboardCardElement.innerHTML = dashboardHTML;
+    return dashboardCardElement;
+  }
+
+  function paginateSectionDashboards(section, dashboardsList, pageSize, pageNumber) {
+    const startIndex = pageNumber * pageSize;
+    const endIndex = (pageNumber + 1) * pageSize;
+    const numberOfPages = Math.ceil(dashboardsList.length / pageSize);
+
+    const sectionDashboardsCardsContainerElement = document.getElementById(
+      section + '-dashboards-cards-container'
+    );
+    sectionDashboardsCardsContainerElement.innerHTML = '';
+    dashboardsList.slice(startIndex, endIndex).forEach(dashboardData => {
+      sectionDashboardsCardsContainerElement.appendChild(buildDashboardCardHTML(dashboardData));
+    });
+
+    const sectionPaginationContainer = document.getElementById(
+      section + '-dashboards-pagination-container'
+    );
+    const pageNumberElement = document.getElementById(section + '-dashboards-current-page');
+    sectionPaginationContainer.style.display = 'none';
+    pageNumberElement.style.display = 'none';
+    if (numberOfPages != 1) {
+      sectionPaginationContainer.style.display = 'flex';
+      const lastPageElement = document.getElementById(section + '-dashboards-last-page');
+      lastPageElement.textContent = numberOfPages.toString();
+
+      const displayedPageNumber = pageNumber + 1;
+      pageNumberElement.textContent = displayedPageNumber.toLocaleString();
+      pageNumberElement.style.display = 'flex';
+    }
+
+    const paginationPrevious = document.getElementById(section + '-dashboards-previous');
+    paginationPrevious.style.display = 'none';
+    if (pageNumber > 0) {
+      paginationPrevious.style.display = 'flex';
+    }
+
+    const paginationNext = document.getElementById(section + '-dashboards-next');
+    paginationNext.style.display = 'none';
+    if (pageNumber < numberOfPages - 1) {
+      paginationNext.style.display = 'flex';
+    }
+    sectionDashboardsCardsContainerElement.dataset.currentPage = pageNumber;
+    sectionDashboardsCardsContainerElement.dataset.lastPage = numberOfPages - 1;
+  }
+
+  function handleFilterBySection(section) {
+    const containers = document.getElementsByName('section-dashboards-container');
+    for (const containerElement of containers) {
+      if (!section) {
+        containerElement.style.display = 'block';
+        continue;
+      }
+
+      if (containerElement === document.getElementById(section + '-dashboards-container')) {
+        containerElement.style.display = 'block';
+      } else {
+        containerElement.style.display = 'none';
+      }
+    }
```

Comment:
A straightforward approach to hiding things that I've used in the past:
- You can use [hidden](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden) instead of display
- "Clearing the board" before setting/resetting hidden has saved me a lot of headache from weird bugs. Recommend setting all to hidden (or not) before then just toggling on/off the one or two you want to show.
- `querySelectorAll` will let you `.forEach` over things


```suggestion
    const containers = document.querySelectorAll('section-dashboards-container');
    containers.forEach(c => c.hidden = !!section);
    if (!section) return;
    containers.querySelector(`#${section}-dashboards-container`).hidden = false;
```